src/framework/uicomponents/qml/Muse/UiComponents/IncrementalPropertyControl.qml

```diff
@@ -30,6 +30,7 @@ Item {
     property int iconMode: !iconImage.isEmpty ? IncrementalPropertyControl.Left : IncrementalPropertyControl.Hidden
     property int iconBackgroundSize: 20
     property alias icon: iconImage.iconCode
+    property alias prefixIcon: textInputField.hintIcon
```

Comment:
I seem to recall overhearing that the designers want to make all `icon`s look like `prefixIcon`s, which would of course allow `IncrementalPropertyControl` to be simplified quite a bit. Might be worth checking if that's still the plan.

Reply:
Yes they do, but I'd prefer to keep it for a future PR